### ISSUE TITLE
Invoices don't have route_hints

### ISSUE THREAD

**Issue by Perlover** [hsjoberg/blixt-wallet#1400]
Hello,

A man billed me from Blixt at iPhone. I couldn't pay for it. Reason: no routes found.  I set up a wallet for him earlier and channels were automatically opened there through the default provider at the first incoming payments. When I decoded the invoice,  `router_hints` was missing. At the same time, the node itself is, of course, private. I have the Android version and router_hints are present in the accounts (but I have manualy created channels with some nodes)

I don't have his phone next to me, but he told me on the phone that he had two channels, they were both active at that moment and the wallet was running. I don't know the details of the channels yet. I can suspect that he did not have enough incoming liquidity in the channels to accept my amount. But since the channels were already open, the wallet apparently decided not to open a new channel through the wallet provider's LSP. It is possible that in such a situation an invoice is created, but `route_hints` are not inserted.

Or it is possible that there was liquidity, but for some reason `route_hints` were not inserted. Although I'm leaning towards the first version and asked the person to send me screenshots of the channels. He's going to do it soon.

In any case, this is a very unpleasant bug, as the user has a deadlock situation. It seems to me that either a new channel should be created automatically, or the user should be informed that he cannot accept funds due to lack of liquidity (I do not see a problem to ask through API the LND inside for remote balances and calculate how much the user can accept the maximum sats)

**Comment by porcupine1178** [comment]
You are right, it ultimately was some other issue. I had a channel with a Blixt node and could make payments to various services, but could not *receive* payments from any service, hence why I jumped the gun on routing issues. I still do not know what the exact issue was, but my current guess is poor networking/routing of the Blixt node, because I have since been able to make and receive payments through channels with another node without any issue.
